app/lib/features/events/widgets/events_list.dart

```diff
@@ -21,43 +21,35 @@ class EventsList extends ConsumerWidget {
-          padding: const EdgeInsets.only(left: 8, top: 5),
-          child: Column(
-            crossAxisAlignment: CrossAxisAlignment.start,
-            children: [
-              events.isNotEmpty
-                  ? Column(
-                      crossAxisAlignment: CrossAxisAlignment.start,
-                      children: <Widget>[
-                        ListView.builder(
-                          shrinkWrap: true,
-                          itemCount: eventsLimit,
-                          physics: const NeverScrollableScrollPhysics(),
-                          itemBuilder: (context, idx) =>
-                              EventItem(event: events[idx]),
-                        ),
-                      ],
-                    )
-                  : Padding(
-                      padding: const EdgeInsets.all(8.0),
-                      child: Text(
-                        'At this moment, you are not joining any upcoming events. To find out what events are scheduled, check your spaces.',
-                        style: Theme.of(context).textTheme.bodySmall,
-                      ),
+        return Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            events.isNotEmpty
+                ? ListView.builder(
+                    shrinkWrap: true,
+                    itemCount: eventsLimit,
+                    physics: const NeverScrollableScrollPhysics(),
+                    itemBuilder: (context, idx) =>
+                        EventItem(event: events[idx]),
+                  )
+                : Padding(
+                    padding: const EdgeInsets.all(8.0),
+                    child: Text(
+                      'At this moment, you are not joining any upcoming events. To find out what events are scheduled, check your spaces.',
+                      style: Theme.of(context).textTheme.bodySmall,
                     ),
-              eventsLimit != events.length
-                  ? Padding(
-                      padding: const EdgeInsets.only(top: 10),
-                      child: OutlinedButton(
-                        onPressed: () =>
-                            context.pushNamed(Routes.calendarEvents.name),
-                        child: Text('See all my ${events.length} events'),
-                      ),
-                    )
-                  : const SizedBox.shrink(),
-            ],
-          ),
+                  ),
+            eventsLimit != events.length
+                ? Padding(
+                    padding: const EdgeInsets.only(top: 10),
+                    child: OutlinedButton(
+                      onPressed: () =>
+                          context.pushNamed(Routes.calendarEvents.name),
+                      child: Text('See all my ${events.length} events'),
+                    ),
+                  )
+                : const SizedBox.shrink(),
+          ],
         );
       },
       loading: () => const CircularProgressIndicator(),
```

Comment:
using a skeleton with a few items similar to the new design would have been nice(r) here ...

Reply:
Added skeleton UI :)
![Frame 1 (1)](https://github.com/acterglobal/a3/assets/51526480/a6c4e7f4-4b6b-4f23-897e-c448f8f9bfc9)

Reply:
oh, so pretty :glowing_eyes: .